src/index.html

```diff
@@ -6,9 +6,28 @@
       name="viewport"
       content="width=device-width,initial-scale=1,maximum-scale=1,user-scalable=0"
     />
-    <link rel="icon" href="https://joincircles.net/wp-content/uploads/2021/10/favicon.png" type="image/png" />
-    <link rel="apple-touch-icon" href="https://joincircles.net/wp-content/uploads/2021/10/favicon.png" />
-    <meta name="msapplication-TileImage" content="https://joincircles.net/wp-content/uploads/2021/10/favicon.png" />
+    <link
+      rel="icon"
+      href="https://joincircles.net/wp-content/uploads/2021/10/favicon.png"
+      type="image/png"
+    />
+    <link
+      rel="apple-touch-icon"
+      href="https://joincircles.net/wp-content/uploads/2021/10/favicon.png"
+    />
+    <meta
+      name="msapplication-TileImage"
+      content="https://joincircles.net/wp-content/uploads/2021/10/favicon.png"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <style>
+      body {
```

Comment:
temporary `body` here, will be moved once we gather styles for specific text types